src/future.cpp

```diff
@@ -183,11 +195,53 @@ struct timed_queue {
     }
 
 };
-
 #endif
 
 /**************************************************************************************************/
 
+#elif STLAB_TASK_SYSTEM == STLAB_TASK_SYSTEM_WINDOWS
+
+class task_system
+{
+    PTP_POOL            _thread_pool;
+
+public:
+
+    task_system() {
+        _thread_pool = CreateThreadpool(nullptr);
```

Comment:
_thread_pool is now unused - we are just using the default thread pool so no need to create.